### PR TITLE
Simplify and de-lint GCE Windows kube-up docs.

### PR DESCRIPTION
**Pull request by jeremyje**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:
Replaces instructions to patch in a forked commit with the proper build instructions. Resolve all markdownlint issues.

**Which issue(s) this PR fixes**:
N/A

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: cluster/gce/windows/README-GCE-Windows-kube-up.md [hunk 1]
```diff
@@ -184,3 +184,9 @@ use the steps below to run K8s e2e tests. These steps are based on
 
 After the test run completes, log files can be found under the `${ARTIFACTS}`
 directory.
+
+## Continuous Integration
```
[the remark this answers is not in the document]
Comment (jeremyje) [reply cluster/gce/windows/README-GCE-Windows-kube-up.md:188]: I am not sure what you mean here. Can you rephase this comment.

[the remark this answers is not in the document]
Comment (yliaog) [reply cluster/gce/windows/README-GCE-Windows-kube-up.md:188]: i mean the test is not run continuously, so it's a bit confusing to have the header ## Continuous Integration

File: cluster/gce/windows/README-GCE-Windows-kube-up.md [hunk 2]
```diff
@@ -44,19 +45,15 @@ release`. However, that builds binaries for all supported platforms, and can be
...
-```
-# Apply https://github.com/yujuhong/kubernetes/commit/27e608a050a997be5ab736a7cdeb29aa68f3b7ee to your tree:
-curl \
-  https://github.com/yujuhong/kubernetes/commit/27e608a050a997be5ab736a7cdeb29aa68f3b7ee.patch | \
-  git apply
-
+```bash
 # Build binaries for both Linux and Windows:
-make quick-release
+KUBE_BUILD_PLATFORMS="linux/amd64 windows/amd64" make quick-release
```
Comment (pjh) [inline cluster/gce/windows/README-GCE-Windows-kube-up.md:50]: Much better!

**Review by pjh — APPROVED** [review]

**Pull request merged**